lotus-soup/compositions/composition-k8s.toml

```diff
@@ -25,7 +25,7 @@
   clients = "1"
   miners = "1"
   genesis_timestamp_offset = "100000"
-  balance = "2000"
+  balance = "20000000"
```

Comment:
These balances will work for maximum 100 nodes, as TotalFilecoin is 2B:

https://github.com/filecoin-project/lotus/blob/3b826e41f6afbcccd2f93c7f3fcd89f1574b4f84/build/params_shared_vals.go#L64

Mind putting a comment here and in all other compositions? Pretty sure if we go above 100 nodes, it'll be a head scratcher and we'll end up burning time debugging why things break.